src/odin/enhancedtrippath.cc

```diff
@@ -122,7 +122,7 @@ const std::string& TripLeg_TravelMode_Name(int v) {
 
 const std::string& TripLeg_VehicleType_Name(int v) {
   static const std::unordered_map<int, std::string> values{
-      {0, "kCar"}, {1, "kMotorcycle"}, {2, "kAutoBus"}, {3, "kTractorTrailer"}, {4, "kMotorScooter"},
+      {0, "kCar"}, {1, "kMotorcycle"}, {2, "kAutoBus"}, {3, "kTruck"}, {4, "kMotorScooter"},
```

Comment:
should update the docs to explain the defaults are in line with what is typically called a semi-trailer truck

Reply:
I guess that’s kinda implicit by the default values for dimensions etc, but wouldn’t hurt to spell it out